README.md

```diff
@@ -59,6 +59,88 @@ style guide that you will not find in the GitHub one.
 
 ## Standards ##
 
+### Naming Convention ###
+
+With Swift 3, Apple updated the Swift [API design guidelines](https://swift.org/documentation/api-design-guidelines/) to promote clarity over brevity.
+
+Capitalize all type names with camel case, while function and variable names start with a lower case. Avoid acronyms and abbreviations for clarity and readability.
+
+#### Protocols ####
+
+Protocols describing something should be a noun: `Collection`, `Element`. Protocols describing an ability should end with “ing” or “able”: `Evaluatable`, `Printable`, `Formatting`.
+
+#### Enum ####
+
+Enum cases should start with a lower case and be camel case.
+
+```
+enum Color {
+    case red
+    case blue
+    case green
+}
+```
+
+#### Functions ####
+
+Include words that can make the function’s behavior clearer. Example of a function that removes an element at an index x.
```

Comment:
Not clear..

How about `You should name your function with words that describe its behavior. Let's take the example of a function that removes and element at a given index 'x':`